cosmo_tester/framework/util.py

```diff
@@ -20,11 +20,15 @@
 import re
 import json
 import shutil
+import string
+import random
+import tempfile
 
 import jinja2
-from path import path
 import yaml
+from path import path
 from itsdangerous import base64_encode
+from jinja2 import Template
```

Comment:
I think it'd be better to import `jinja2` and do `jinja2.Template` where necessary